### ISSUE TITLE
Image thumbnail size 

### ISSUE THREAD

**Issue by Jeremy8776** [hayden-fr/ComfyUI-Model-Manager#81]
This happens quite a lot

![image](https://github.com/user-attachments/assets/0f3b6aeb-f2a5-43f6-aea1-d7e1bba025d0)

**Comment by hayden-fr** [comment]
Indeed, this issue has occurred for many users in the past, but it has never happened on my end.

I suspect it may be related to the resolution.

I would be very grateful if you could provide some information to help me fix it.

**Comment by Jeremy8776** [comment]
1440p monitor, using chrome for browser

**Comment by vitosans** [comment]
@hayden-fr  - What info would be good to help debug this?

**Comment by hayden-fr** [comment]
After I removed some css features it seemed to reproduce the issue. Please consider updating your browser version to try to resolve this issue.

**Comment by Jeremy8776** [comment]
> After I removed some css features it seemed to reproduce the issue. Please consider updating your browser version to try to resolve this issue.

Chrome is always up-to-date.

**Comment by vitosans** [comment]
yeah, still happening here. Let me know what debug I can provide / dumps and I will be happy to assist.

**Comment by hayden-fr** [comment]
Such as OS version, browser version, ComfyUI version, ComfyUI_frontend version, enabled ComfyUI custom_nodes

**Comment by sansmoraxz** [comment]
> Such as OS version, browser version, ComfyUI version, ComfyUI_frontend version, enabled ComfyUI custom_nodes

Might I suggest having issue templates.

**Comment by hayden-fr** [comment]
> Might I suggest having issue templates.

Yes, I think you can refer to [ComfyUI_frontend/bug-report](https://github.com/Comfy-Org/ComfyUI_frontend/blob/main/.github/ISSUE_TEMPLATE/bug-report.yaml)
